tests/039_worker_deno_ns.ts

```diff
@@ -0,0 +1,25 @@
+const w1 = new Worker("./tests/039_worker_deno_ns/has_ns.ts");
+const w2 = new Worker("./tests/039_worker_deno_ns/no_ns.ts", {
+  deno: { noDenoNamespace: true }
```

Comment:
This API seems a bit convoluted. What if it was just:

Worker(x, { noDenoNamespace: true })

Reply:
Done.